### DISCUSSION TITLE
Eksterne lenker

### DISCUSSION THREAD

**Discussion by mimarz** [digdir/designsystemet#1977] [Patterns for public services]
How do we define external links?

[Examples](https://codesandbox.io/p/sandbox/ekstern-lenke-8fgcr6?file=%2Fsrc%2FApp.tsx%3A11%2C27)

With "external link icon" or not?
Descriptive that its an external link?
Always open in new tab or not? 

[UK Gov does not recommend using icons for external icons](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/)

**Comment by jeffreiffers** [comment]
When is a link an external link?

1. Link opens a new tab (target="_blank")
2. Link to other domain/website

We also need some best practices when to use target="_blank" and when not to use.

**Comment by mrosvik** [comment]
Duplicate of https://github.com/digdir/designsystemet/discussions/2294
